com.visualstudio.code.insiders.appdata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
huh, this file should have a launchable tag

https://github.com/flathub/flathub/wiki/AppData-Guidelines

`<launchable type="desktop-id">com.visualstudio.code.insiders.desktop</launchable>`